talk/owt/sdk/p2p/p2ppeerconnectionchannel.cc

```diff
@@ -1297,12 +1297,15 @@ void P2PPeerConnectionChannel::DrainPendingRemoteCandidates() {
   rtc::scoped_refptr<webrtc::PeerConnectionInterface> temp_pc_ = GetPeerConnectionRef();
   {
     rtc::CritScope cs(&pending_remote_candidates_crit_);
-    for (auto& ice_candidate : pending_remote_candidates_) {
-      if (!temp_pc_->AddIceCandidate(ice_candidate.get())) {
-        RTC_LOG(LS_WARNING) << "Failed to add remote candidate.";
+    if (temp_pc_->remote_description()) {
+      RTC_LOG(LS_INFO) << "Draining pending ICE Candidates, received " << pending_remote_candidates_.size();
+      for (auto& ice_candidate : pending_remote_candidates_) {
```

Comment:
Could this be `const auto` ?

Reply:
Yeah, I suppose I can. Won't hurt and might catch a bug in the future.

Reply:
Thanks! I'm usually super-anal with regard to const correctness. ;)